release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -589,6 +589,13 @@ The Self Node Remediation Operator provides the following enhancements:
 
 For more information, see xref:../nodes/nodes/eco-self-node-remediation-operator.adoc#self-node-remediation-operator-remediate-nodes[Remediating nodes with the Self Node Remediation Operator].
 
+[id="ocp-4-11-nodes-worker-nodes"]
+==== Worker nodes for {sno} clusters
+
+You can now add worker nodes to {sno} clusters. This is useful for deployments in constrained environments or at the network edge when you need to add additional capacity to your cluster.
```

Comment:
is it **constrained or resource-constrained** environment?  When I looked at the OCP docs, I could see instances of resource-constrained.

Reply:
Updated.